cli/src/utilities/runCLI.ts

```diff
@@ -64,6 +104,57 @@ export async function runCLI(toolbox: Toolbox, projectName: string): Promise<Cli
         }
       });
     }
+    // If the user is using a version of bun that is anything but version 1.0.14, communicate a message
+    if (
+      cliResults.flags.packageManager === 'bun' &&
+      defaultPackageManagerVersion &&
+      defaultPackageManagerVersion !== recommendedBunVersion
```

Comment:
If the user's default package manager is `yarn`, is this going to match up after they chose via cli prompt to use `bun`?

Reply:
yea this kinda borks if you choose another package manager, it still reads `recommendedBunVersion` and says some shit like "bet, we're gonna use yarn v20.x.x" when it actually originally read the version from npm. not sure if that's what you're referring to.